docs/.vuepress/styles/index.styl

```diff
@@ -0,0 +1,7 @@
+hr {
+  border-top: 1px solid #000000;
+}
+blockquote {
```

Comment:
Do we want to adjust _all_ of the block quotes? For example, now the actual quotes look a little funky. https://deploy-preview-2820--pomerium-docs.netlify.app/docs/background.html

Reply:
It might just be easier to not have those items in a quote block. It looks fine to me just as bullets. 

<img width="708" alt="Screen Shot 2021-12-30 at 11 49 31 AM" src="https://user-images.githubusercontent.com/1544881/147783703-d30727aa-c29b-429b-8197-95609c609f44.png">